render_utils.py

```diff
@@ -1,10 +1,15 @@
+import imageio
 import nerf
 import numpy as np
+import os
 import torch
 import utils
 
 import torch.nn.functional as F
 
+#Converts an image's pixels values from [0, 1] to [0, 255]
+to8b = lambda x : (255*np.clip(x,0,1)).astype(np.uint8)
```

Comment:
nitpick: isn't this already defined in `utils`? you might just want to use `utils.to8b` or just define it here

Reply:
lol, this was just me not knowing i could access a lambda function from another module... that is def the right call